lms/djangoapps/courseware/tabs.py

```diff
@@ -153,7 +153,7 @@ class TextbookTabs(TextbookTabsBase):
     A tab representing the collection of all textbook tabs.
     """
     type = 'textbooks'
-    priority = None
+    priority = 200
```

Comment:
This is a tab collection. So I've given it a 3-digit number potentially allowing us to have each child tab have a priority in the range of 202-300. It's unlikely a course will have more than a few of these.